pinot-tools/src/main/resources/conf/pinot-broker-log4j2.xml

```diff
@@ -39,6 +44,10 @@
       <!-- Direct most logs to the log file -->
       <AppenderRef ref="brokerLog"/>
     </Root>
+    <!-- Output query logs to a dedicated file-->
```

Comment:
I don't see pinot-broker-log4j2.xml being used in any pom.xml (maybe I'm missing it). Did we check if this resulted in a separate log file for Querylogger?